src/ontology/uberon-edit.obo

```diff
@@ -3341,7 +3341,7 @@ relationship: only_in_taxon NCBITaxon:33213 ! Bilateria
 id: UBERON:0000154
 name: posterior region of body
 is_a: UBERON:0000475 ! organism subdivision
-relationship: only_in_taxon NCBITaxon:33213 ! Bilateria
+relationship: only_in_taxon NCBITaxon:7742 ! Vertebrata <vertebrates>
```

Comment:
Again: please add the taxon restriction directly to the new term _paraspinal region_ that you are adding in this PR, not to any parental term.